package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "next-sitemaps-plugin",
+  "version": "1.0.0",
+  "description": "A plugin to create sitemaps in a next application",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "jest --coverage",
```

Comment:
Please remove the coverage flag here.

Reply:
We are also missing CI/CD. Either use Travis or Github actions. You may check other repos for examples.